openrr-apps/config/sample_robot_client_config_for_plugin.toml

```
@@ -42,5 +44,20 @@ args = """
 """
 
 [[plugins.plugin1.instances]]
-name = "move_base"
+name = "plugin1"
+type = "MoveBase"
+
+[[plugins.plugin1.instances]]
+# If the instances are of different types, the names of the instances will not conflict.
+name = "plugin1"
+type = "Navigation"
+
+[plugins.arci_ros2]
+path = "../../target/debug/libarci_ros2"
+
+[[plugins.arci_ros2.instances]]
+name = "arci_ros2"
 type = "MoveBase"
+args = """
+topic = "/demo/cmd_demo"
```

I think we can use `/cmd_vel`, and I feel it's better as a sample.

```suggestion
topic = "/cmd_vel"
```

---

done